tests/kernel/semaphore/semaphore/src/main.c

```diff
@@ -21,6 +21,7 @@
 #define TOTAL_THREADS_WAITING (5)
 
 #define SEC2MS(s) ((s) * 1000)
+#define QSEC2MS(s) ((s) * 250)
```

Comment:
Why is this changing units?  Seems like this needs an explanation.

Reply:
Hi, @andyross , I change this because I got the following message when I run this case.
```
===================================================================
START - test_sem_multiple_take_and_timeouts

    Assertion failed at WEST_TOPDIR/zephyr/subsys/testsuite/ztest/src/ztest.c:125: cpu_hold: (dt < 3000 is false)
1cpu test took too long (5010 ms)
 FAIL - test_sem_multiple_take_and_timeouts in 5.7 seconds
===================================================================
```
So I try to make the time unit smaller to avoid this and save some execution time (without affecting the test case result)

```
===================================================================
START - test_sem_multiple_take_and_timeouts
 PASS - test_sem_multiple_take_and_timeouts in 1.255 seconds
===================================================================

```
Thank you!